practices/practice-two/src/styles/common/display.css

```diff
@@ -0,0 +1,35 @@
+/*
+mobile devices - sm
+ */
+@media screen and (min-width: 320px) and (max-width: 480px) {
```

Comment:
For breakpoints, please create it as a variable

Reply:
I can't use variables